shell/common/animator.h

```diff
@@ -106,15 +106,15 @@ class Animator final {
 
   std::unique_ptr<FrameTimingsRecorder> frame_timings_recorder_;
   uint64_t frame_request_number_ = 1;
-  int64_t dart_frame_deadline_;
+  int64_t dart_frame_deadline_ = 0;
```

Comment:
Can we convert this to use time points in FML instead? I can't tell what unit the deadline is supposed to be in.

I know this wasn't the point of this patch but moving the initializers to the header seems to be a drive-by change. So, perhaps you are interested in cleaning this little but up as well. Fine if you'd rather not.

Reply:
done - would you mind taking a look to make sure what I did looks reasonable?

Reply:
https://github.com/flutter/engine/pull/29007/commits/8c2e7a7543d80432743e0273ca1d29d86ac9c116